### DISCUSSION TITLE
SteamOS Support

### DISCUSSION THREAD

**Discussion by thereisnoh** [bitfocus/companion#2902] [Ideas]
Any possibility of a SteamOS compatible version? Isn't it just a specialized Linux distro? It would be very nice to easy run Companion on my Steam Deck with a Stream Deck attached (unfortunate naming similarity).

**Comment by Julusian** [comment]
Do you know what would be needed to achieve this?
I don't have a steam deck, so have no experience with how steamos is different to other linux or whether the existing linux packages would work

**Comment by thereisnoh** [comment]
Not entirely. Ver 2.0 worked unmodified and after ver 3 was released it no
longer launches.

On Sat, Jun 8, 2024 at 12:12 PM Julian Waller ***@***.***>
wrote:

> Do you know what would be needed to achieve this?
> I don't have a steam deck, so have no experience with how steamos is
> different to other linux or whether the existing linux packages would work
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bitfocus/companion/discussions/2902#discussioncomment-9714910>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACWQWA46EWI62CONUDWDBYTZGMUPHAVCNFSM6AAAAABI64VFH6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TOMJUHEYTA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Jon Little

**Reply by Julusian** [reply]
Does it log anything that might indicate what is wrong?

A lot changed in 3.0 about how companion is built
